dpdata/plugins/deepmd.py

```diff
@@ -136,6 +136,7 @@ def mix_system(self, *system, type_map, **kwargs):
 
     def from_multi_systems(self, directory, **kwargs):
         sys_dir = []
+        assert os.path.isdir(directory), f"{directory} is not a directory!"
```

Comment:
Please raise `FileNotFoundError`. One has not chance of catching the error with assertion! 
Plz also add a UT for it.